insomniac.py

```diff
@@ -78,10 +78,15 @@ def main():
             print("Action: remove " + str(args.remove_mass_followers) + " mass followers")
             mode = Mode.REMOVE_MASS_FOLLOWERS
 
+    args.interactions_count = get_count(args.interactions_count, 'Interactions count', 70)
```

Comment:
Why do we need these lines?
It's better not to touch args, as they are our script's input.

Reply:
Btw calculating `get_count()` on this stage is the same as to just pass some another value to the script, isn't it?